frontend/js/actions/setBattle.js

```diff
@@ -0,0 +1,12 @@
+import { getTeamData } from 'utils/api';
+
+import { BATTLE_DETAIL } from '../constants';
+
+function fetchBattle(battle) {
```

Comment:
Also the names are inconsistent: `fetchBattle`, `getBattleList`, and `setCurrentUser`